Blish HUD/GameServices/Gw2WebApi/ManagedConnection.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Gw2Sharp;
+using Gw2Sharp.WebApi;
+using Gw2Sharp.WebApi.Caching;
+using Gw2Sharp.WebApi.V2.Models;
+
+namespace Blish_HUD.Gw2WebApi {
+    public sealed class ManagedConnection {
+
+        private static readonly Logger Logger = Logger.GetLogger<ManagedConnection>();
+
+        private readonly Connection _internalConnection;
+
+        public IConnection Connection => _internalConnection;
+
+        private readonly IGw2WebApiClient _internalClient;
+
+        public IGw2WebApiClient Client => _internalClient;
+
+        public ManagedConnection(string accessToken, ICacheMethod webApiCache, ICacheMethod renderCache = null, TimeSpan? renderCacheDuration = null) {
+            string ua = $"BlishHUD/{Program.OverlayVersion} ";
```

Comment:
I think there is an excess whitespace in there, or is it intentional? Why?

Reply:
This was due to this:
https://discordapp.com/channels/384735285197537290/589031897963692034/692548369742168084

> So taking a closer look at the UA appending that is done, a space is not placed in between.  This causes the UA format to be incorrect (I believe due to having multiple / without being split by a  ).  Appending my provided UA with a space allows it to work.

It appears that this was fixed in 9.2, though, so I'll get that extra space removed. :)

https://github.com/Archomeda/Gw2Sharp/commit/022d7296cc3dc64c883116105987a626136edc74